pyfolio/utils.py

```diff
@@ -44,6 +47,10 @@
     MONTHLY: MONTHS_PER_YEAR
 }
 
+DEPRECATION_WARNING = ("Data loaders in pyfolio.utils are deprecated "
+                       "and will be removed in a future release. Please "
+                       "install the empyrical package instead.")
```

Comment:
"Data loaders have been moved to empyrical and will be removed from pyfolio in a future release. Please use e.g. empyrical.utils.get_symbol_rets() instead of pyfolio.utils.get_symbol_rets()."